shells/firefox/build.js

```diff
@@ -25,13 +25,13 @@ const main = async () => {
   console.log(chalk.gray('\n# Go to the unpacked directory:'));
   console.log(`cd ${unpackedPath}`);
   console.log(chalk.gray('\n# And launch Firefox with the extension enabled:'));
-  console.log('web-ext run');
+  console.log('yarn run start:firefox');
```

Comment:
Oh right this is why I created the `start:firefox`.  I wasn't sure what to do here because now that we're installing web-ext this would need to change to be `../../node_modules/.bin/web-ext` or I can give instructions using the yarn script from the root directory; something like `(cd ../../ && yarn run start:firefox)`.  I think I'll give the location for `web-ext`.